docs/developer-docs/integrations/rosetta/index.md

```diff
@@ -8,68 +10,67 @@ This documentation might help if you want to deploy a token on the IC that aims
 
-    ``` bash
-    docker pull dfinity/rosetta-api
-    ```
+### Step 2:  Pull the latest `dfinity/rosetta-api` image from the Docker Hub by running the following command:
 
-3.  Start the integration software by running the following command:
-
-    ``` bash
-    docker run \
-        --interactive \
-        --tty \
-        --publish 8080:8080 \
-        --rm \
-       dfinity/rosetta-api
-    ```
+``` bash
+docker pull dfinity/rosetta-api
+```
 
-    This command starts the software on the local host and displays output similar to the following:
+### Step 3:  Start the integration software by running the following command:
 
-        Listening on 0.0.0.0:8080
-        Starting Rosetta API server
+``` bash
+docker run \
+    --interactive \
+    --tty \
+    --publish 8080:8080 \
+    --rm \
+    dfinity/rosetta-api
+```
 
-    By default, the software connects to a testnet.
-    It **does not** connect to the ledger canister running on the Internet Computer blockchain mainnet.
+This command starts the software on the local host and displays output similar to the following:
 
-    If you have been assigned a test network and corresponding ledger canister identifier, you can run the command against that network by specifying an additional `canister` argument.
-    For example, the following command illustrates connecting to the ledger canister on a test network by setting the `canister` argument to `2xh5f-viaaa-aaaab-aae3q-cai`.
+    Listening on 0.0.0.0:8080
+    Starting Rosetta API server
 
-    ``` bash
-    docker run \
-        --interactive \
-        --tty \
-        --publish 8080:8080 \
-        --rm \
-       dfinity/rosetta-api \
-       --canister 2xh5f-viaaa-aaaab-aae3q-cai
-    ```
+By default, the software connects to a testnet.
+It **does not** connect to the ledger canister running on the Internet Computer blockchain mainnet.
 
-    :::note
+If you have been assigned a test network and corresponding ledger canister identifier, you can run the command against that network by specifying an additional `canister` argument.
+For example, the following command illustrates connecting to the ledger canister on a test network by setting the `canister` argument to `2xh5f-viaaa-aaaab-aae3q-cai`.
 
-    The first time you run the command, it might take some time for the node to catch up to the current link of the chain.
-    When the node is caught up, you should see output similar to the following:
+``` bash
+docker run \
+    --interactive \
+    --tty \
+    --publish 8080:8080 \
+    --rm \
+    dfinity/rosetta-api \
+    --canister 2xh5f-viaaa-aaaab-aae3q-cai
+```
 
-    :::
+:::info
+The first time you run the command, it might take some time for the node to catch up to the current link of the chain.
+When the node is caught up, you should see output similar to the following:
+:::
 
-        You are all caught up to block height 109
+    You are all caught up to block height 109
```

Comment:
I believe this message belongs the "info" block above.

Reply:
Good catch, thank you!